infrastructure/template.yaml

```diff
@@ -61,15 +79,23 @@ Resources:
           Value: !Ref 'Environment'
         - Key: Source
           Value: "https://github.com/alphagov/di-account-management-backend/blob/main/infrastructre/template.yaml"
+      PointInTimeRecoverySpecification:
+              PointInTimeRecoveryEnabled: true
+      SSESpecification:
+        SSEEnabled: true
+        SSEType: KMS
+        KMSMasterKeyId: !GetAtt DatabaseKmsKey.Arn
 
   ######################################
   # TxMA Event Queue (Dummy for now)
   ######################################
   TxMAInputDummyQueue:
     Type: AWS::SQS::Queue
+    Properties:
+      KmsMasterKeyId: !GetAtt QueueKmsKey.Arn
     DeletionPolicy: Delete
     UpdateReplacePolicy: Delete
-
+    
```

Comment:
Ok really non blocking but might want to clear out that whitespace there too.

I think Alex and I both have trim whitespace on here. I think if I saved the file it'd auto edit this line too, which might be a bit odd for any future reviewers.

Here's where I've got it turned on:
![image](https://user-images.githubusercontent.com/3694062/199057642-ce382ca2-0b40-455a-8a4d-26c26dfed05e.png)